pom.xml

```diff
@@ -49,6 +49,9 @@ THE SOFTWARE.
     <aetherVersion>0.9.0.M3</aetherVersion>
     <sisuInjectVersion>0.0.0.M5</sisuInjectVersion>
     <wagonVersion>2.4</wagonVersion>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+    <!--TODO: change to true after the code cleanup-->
+    <maven.findbugs.failure.strict>false</maven.findbugs.failure.strict>
```

Comment:
I've improved the pattern a bit. I hope to streamline the implementation when and if we start implementing these features on the parent pom.